bigchaindb/commands/bigchaindb.py

```diff
@@ -124,21 +128,28 @@ def run_upsert_validator_new(args, bigchain):
     """
 
     new_validator = {
-        'public_key': args.public_key,
+        'public_key': public_key_from_base64(args.public_key),
         'power': args.power,
         'node_id': args.node_id
     }
 
-    key = load_node_key(args.sk)
-
-    voters = ValidatorElection.recipients(bigchain)
+    try:
+        key = load_node_key(args.sk)
+        voters = ValidatorElection.recipients(bigchain)
+        election = ValidatorElection.generate([key.public_key],
+                                              voters,
+                                              new_validator, None).sign([key.private_key])
+        election.validate(bigchain)
+    except ValidationError as e:
```

Comment:
We return `False` when some errors occur and raise an exception in some other cases. Should we make it consistent?

Reply:
In case of exceptions during execution of the command the return value is `False` and suitable log messages are generated for the users.